tests/storage/redis/merge_reader.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""Tests for the merge reader for Redis stores."""
+from __future__ import unicode_literals
+
+import os
+
+import fakeredis
+import redis
+
+from plaso.containers import sessions
+from plaso.containers import tasks
+from plaso.lib import definitions
+from plaso.storage.redis import merge_reader
+from plaso.storage.redis import redis_store
+from plaso.storage.redis import writer as redis_writer
+from plaso.storage.sqlite import writer
+
+from tests import test_lib as shared_test_lib
+from tests.containers import test_lib as containers_test_lib
+from tests.storage import test_lib
+
+class RedisMergeReaderTest(test_lib.StorageTestCase):
+  """Tests for the Redis store reader for merging."""
+
+  def _GetRedisClient(self):
+    """Creates a Redis client for testing.
+
+    This method will attempt to use the
+
+    Returns:
+      Redis: a Redis client.
+    """
+    try:
+      redis_client = redis.from_url('redis://127.0.0.1/0', socket_timeout=60)
+      redis_client.ping()
+    except redis.exceptions.ConnectionError:
+      redis_client = fakeredis.FakeStrictRedis()
+
+    return redis_client
+
+  def _CreateTaskStore(self, session, redis_client):
+    """Creates a task store for testing.
+
+    Args:
+      session (Session): session the task store is part of.
+      redis_client (Optional[Redis]): Redis client to query. If specified, no
```

Comment:
please remove Optional, argument is a positional arg

Reply:
Done